docs/baking.md

```diff
@@ -337,3 +337,12 @@ sudo systemctl restart tezos-node-$tznet
 ```
 
 </details>
+
+## Protocol switch-over
```

Comment:
This is kind of an odd (and not very visible) position to put this section IMO.

I'd move it to be a subsection of the `Installation` one instead, since it's mostly about updates to the package itself. What do you think?

Reply:
Yeah, I agree that it makes sense. Though, I've renamed the section itself